### PR TITLE
feat: decrease retry timeouts [fixes DXJ-764]

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: src/lib/deal.ts [hunk 1]
```diff
@@ -205,9 +205,11 @@ export async function match(dealAddress: string) {
       return dealMatcherClient.getMatchedOffersByDealId(dealAddress);
     },
     (err) => {
-      commandObj.error(stringifyUnknown(err));
+      commandObj.error(
+        `Wasn't able to find a match for the deal ${dealAddress}: ${stringifyUnknown(err)}`,
+      );
     },
-    1000 * 60 * 2, // 2 minutes
+    1000 * 5,
```
Comment (folex) [inline src/lib/deal.ts:212]:
```suggestion
    1000 * 5 // 5 seconds,
```

File: src/lib/dealClient.ts [hunk 2]
```diff
@@ -140,9 +140,11 @@ async function createDealClient(
       await core.minDealDepositedEpochs();
     },
     (err) => {
-      throw new Error(stringifyUnknown(err));
+      commandObj.error(
+        `Check if blockchain client is connected by running core.minDealDepositedEpochs() failed: ${stringifyUnknown(err)}`,
+      );
     },
-    1000 * 60 * 5,
+    1000 * 5,
```
Comment (folex) [inline src/lib/dealClient.ts:147]:
```suggestion
    1000 * 5 // 5 seconds,
```

File: src/lib/dealClient.ts [hunk 3]
```diff
@@ -255,11 +257,15 @@ export async function sign<T extends unknown[]>(
     (err) => {
       throw err;
     },
-    1000 * 60 * 3,
+    1000 * 5,
```
Comment (folex) [inline src/lib/dealClient.ts:260]:
```suggestion
    1000 * 5, // 5 seconds
```

**Review by folex — APPROVED** [review]

File: src/lib/dealClient.ts
Comment (folex) [inline src/lib/dealClient.ts:1]: What about number of retries while submitting a TX? How to control it?

Comment (shamsartem) [reply src/lib/dealClient.ts:1]: retry happens every second so it most likely will be couple of them. If we are intersted specifically in trying fixed number of times - we should improve this setTryTimeout function

Comment (folex) [reply src/lib/dealClient.ts:1]: Let's see how it goes. I just wasn't sure it applies only to `minDealDepositedEpochs` or to any transaction

**Pull request merged**